scripts/setup/register-server

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+EXTERNAL_HOST="zulip.example.com"
+ZULIP_ADMINISTRATOR="zulip-admin@example.com"
+zulip_org_id=2
+zulip_org_key=2
+
+if [ "$EXTERNAL_HOST" != "zulip.example.com" ]; then
+    echo "Please set EXTERNAL_HOST in /etc/zulip/settings.py."
+    exit
+fi
+
+if [ "$ZULIP_ADMINISTRATOR" != "zulip-admin@example.com" ]; then
+    echo "Please set ZULIP_ADMINISTRATOR in /etc/zulip/settings.py."
+    exit
+fi
+
+if [[  -z "$zulip_org_id" ]] ; then
+    echo  "Please run scripts/setup/generate_secrets.py first."
+    exit
+fi
+if [[ -z "$zulip_org_key"  ]]; then
+    echo  "Please run scripts/setup/generate_secrets.py first."
+    exit
+fi
+
+echo "This script will register your server with the Zulip push notification bouncer service."
+echo  " You can read more about the service at 
+    http://zulip.readthedocs.io/en/latest/prod-mobile-push-notifications.html. "
+RESPONSE="n"
+
+while [[ $RESPONSE =~ ^[nN]$ ]];
+do
+read -p 'Hostname (EXTERNAL_HOST): ' EXTERNAL_HOST
+read -p 'Contact email (ZULIP_ADMINISTRATOR): ' ZULIP_ADMINISTRATOR
+read -p 'Is the information above correct? We recommend using an email address 
+like ops@ that will outlast changes in personnel. [Y/n] ' RESPONSE
```

Comment:
This section should be moved up before the push notification bouncer text.

Reply:
ah, I see where you got this text :). Okay, let's instead just move
```
Use of the bouncer is subject to the Zulip Terms of Service at https://zulipchat.com/terms/.
Do you agree to those terms? [Y/n]
```
to right after 
```
This script will register your server with the Zulip push notification bouncer service.
You can read more about the service at 
    http://zulip.readthedocs.io/en/latest/prod-mobile-push-notifications.html. 
```

Thanks!

Reply:
@rishig Moved the text.